README.md

```diff
@@ -62,6 +62,7 @@ Signed,
 - Ken Wheeler ([@kenwheeler](https://github.com/kenwheeler)), maintainer of Slick
 - Kent C. Dodds ([@kentcdodds](https://github.com/kentcdodds)), maintainer of angular-formly
 - Mario Zechner ([@badlogicgames](https://github.com/badlogic)), maintainer of libGDX
+- Merrick Christensen ([@iammerrick](https://github.com/iammerrick)), garbage developer
```

Comment:
It means I'm a trash developer, just trying to get by in this industry of fast moving pace. I can change it to something less self deprecating if you're concerned it will hinder the purpose of the letter.